application/src/main/java/org/opentripplanner/apis/vectortiles/DebugVectorTilesResource.java

```diff
@@ -49,8 +49,8 @@
  * Slippy map vector tile API for rendering various graph information for inspection/debugging
  * purposes.
  */
-@Path("/routers/{ignoreRouterId}/inspector/vectortile")
-public class GraphInspectorVectorTileResource {
+@Path("/debugvectortiles")
```

Comment:
Should this be `/debug/vectortiles`. It will allow us to add other endpoints related to debugging OTP under `/debug`and we can in the doc simply state that everything under `/debug` is subject to change without notice.

Reply:
I also considered `/debugui/vectortiles` - but I do not like giving something a named based on the consumer - it might be used by another consumer in the future.

Reply:
I would be okay with it (I even like it better), but @optionsome specifically requested the other path.

If he agrees I will change it.

Reply:
We agreed that we will add the slash.